Example/IntegrationTests/Pairing/PairingTests.swift

```diff
@@ -1,149 +1,149 @@
-
-        let networkingClient = NetworkingClientFactory.create(
-            relayClient: relayClient,
-            logger: logger,
-            keychainStorage: keychain,
-            keyValueStorage: keyValueStorage)
-
-        let pairingClient = PairingClientFactory.create(
-            logger: logger,
-            keyValueStorage: keyValueStorage,
-            keychainStorage: keychain,
-            networkingClient: networkingClient)
-
-        return pairingClient
-    }
-
-    func testProposePushOnPairing() async {
-        let expectation = expectation(description: "propose push on pairing")
-
-        (appPairingClient, appPushClient) = makeClients(prefix: "🤖 App")
-        (walletPairingClient, walletPushClient) = makeClients(prefix: "🐶 Wallet")
-
-        walletPushClient.proposalPublisher.sink { _ in
-            expectation.fulfill()
-        }.store(in: &publishers)
-
-        let uri = try! await appPairingClient.create()
-
-        try! await walletPairingClient.pair(uri: uri)
-
-        try! await appPushClient.propose(topic: uri.topic)
-
-        wait(for: [expectation], timeout: InputConfig.defaultTimeout)
-    }
-
-    func testPing() async {
-        let expectation = expectation(description: "expects ping response")
-
-        (appPairingClient, appPushClient) = makeClients(prefix: "🤖 App")
-        (walletPairingClient, walletPushClient) = makeClients(prefix: "🐶 Wallet")
-
-        let uri = try! await appPairingClient.create()
-        try! await walletPairingClient.pair(uri: uri)
-        try! await walletPairingClient.ping(topic: uri.topic)
-        walletPairingClient.pingResponsePublisher
-            .sink { topic in
-                XCTAssertEqual(topic, uri.topic)
-                expectation.fulfill()
-            }.store(in: &publishers)
-        wait(for: [expectation], timeout: InputConfig.defaultTimeout)
-    }
-
-    func testResponseErrorForMethodUnregistered() async {
-        (appPairingClient, appPushClient) = makeClients(prefix: "🤖 App")
-        walletPairingClient = makePairingClient(prefix: "🐶 Wallet")
-
-        let expectation = expectation(description: "wallet responds unsupported method for unregistered method")
-
-        appPushClient.responsePublisher.sink { (_, response) in
-            XCTAssertEqual(response, .failure(WalletConnectPairing.PairError(code: 10001)!))
-            expectation.fulfill()
-        }.store(in: &publishers)
-
-        let uri = try! await appPairingClient.create()
-
-        try! await walletPairingClient.pair(uri: uri)
-
-        try! await appPushClient.propose(topic: uri.topic)
-
-        wait(for: [expectation], timeout: InputConfig.defaultTimeout)
-
-    }
-
-    func testDisconnect() {
-        // TODO
-    }
-}
+//import Foundation
+//import XCTest
+//import WalletConnectUtils
+//@testable import WalletConnectKMS
+//import WalletConnectRelay
+//import Combine
+//import WalletConnectNetworking
+//@testable import WalletConnectPush
```

Comment:
why PairingTests removed?

Reply:
just realised that, will commit fix in a minute

Reply:
uncommented and fixed